Steamworks/src/org/usfirst/frc/team4786/robot/Robot.java

```diff
@@ -33,7 +32,7 @@
 	@Override
 	public void robotInit() {
 		oi = new OI();
-		chooser.addDefault("Default Auto", new ExampleCommand());
+//		chooser.addDefault("Default Auto", new ExampleCommand());
```

Comment:
Remove the commented out code for the chooser. Actually, remove this entirely. We can re add it back later when we have multiple auto modes.